caravel/assets/javascripts/components/CopyToClipboard.jsx

```diff
@@ -0,0 +1,102 @@
+import React, { PropTypes } from 'react';
+import { Button, Tooltip, OverlayTrigger } from 'react-bootstrap';
+
+const propTypes = {
```

Comment:
I need to replicate this pattern of setting the propTypes & defaultTypes at the top of the file in SqlLab!